lib/Ix/App.pm

```diff
@@ -104,7 +113,13 @@ sub to_app ($self) {
 
     my $ctx;
     my $res = try {
-      $ctx = $self->processor->context_from_plack_request($req);
+      unless ($schema_cache{$$}) {
+        %schema_cache = ($$ => $self->processor->schema_connection);
```

Comment:
So if we fork we obliterate the cache and get a new connection for our new process. I think this is right.